src/examples/extended_scorecard.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+
+from krisi.evaluate import Metric, MetricCategories, SampleTypes, ScoreCard
+
+sc = ScoreCard(
+    model_name="<your_model_name>",
+    dataset_name="<your_dataset_name>",
+    sample_type=SampleTypes.insample,
+)
+
+target, predictions = np.random.rand(100), np.random.rand(100)
+
+""" Predefined metrics """
+for metric in sc.get_default_metrics():
+    metric.evaluate(target, predictions)
+
+""" Adding a new metric """
+calculated_metric_example = (target - predictions).mean()
+
+sc["own_metric"] = calculated_metric_example
+# or
+sc.another_metric = calculated_metric_example * 2.0
+# or
+sc["yet_another_metric"] = Metric(
+    name="A new, own Metric",
+    category=MetricCategories.residual,
+    result=calculated_metric_example * 3.0,
+    hyperparameters={"hyper_1": 5.0},
```

Comment:
by the way, we should consider using the word 'parameters'. these are parameters to a function, hyperparameters are exclusively used in the context of deep learning (which is not very popular in TS, for a reason) where the term "parameters" are already in use.

Reply:
You are right, I can change it.